gensim/models/base_any2vec.py

```diff
@@ -230,7 +354,37 @@ def _train_epoch(self, data_iterable, cur_epoch=0, total_examples=None,
 
     def train(self, data_iterable, epochs=None, total_examples=None,
               total_words=None, queue_factor=2, report_delay=1.0, callbacks=(), **kwargs):
-        """Handle multi-worker training."""
+        """Train the model for every epochs using multiple workers.
+
+        Parameters
+        ----------
+        data_iterable : iterable of list of object
+            The input corpus. This will be split in chunks and these chunks will be pushed to the queue.
+        epochs : int, optional
+            Number of epochs (training iterations over the whole input) of training.
+        total_examples : int, optional
+            Count of objects in the `data_iterator`. In the usual case this would correspond to the number of sentences
+            in a corpus. Used to log progress.
+        total_words : int, optional
+            Count of total objects in `data_iterator`. In the usual case this would correspond to the number of raw
+            words in a corpus. Used to log progress.
+        queue_factor : int, optional
+            Multiplier for size of queue -> size = number of workers * queue_factor.
+        report_delay : float, optional
+            Number of seconds between two consecutive progress report messages in the logger.
+        callbacks : list of :class: `~gensim.models.callbacks.CallbackAny2Vec`, optional
```

Comment:
No need to add space between `:class:` and link to model, please check all in rendered documentation (how this looks), this is important